docker/test/upgrade/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# rebuild in #33610
+# docker build -t clickhouse/upgrade-check .
+ARG FROM_TAG=latest
+FROM clickhouse/stateful-test:$FROM_TAG
+
+RUN apt-get update -y \
+    && env DEBIAN_FRONTEND=noninteractive \
+        apt-get install --yes --no-install-recommends \
+            bash \
+            tzdata \
+            fakeroot \
+            debhelper \
+            parallel \
+            expect \
+            python3 \
+            python3-lxml \
+            python3-termcolor \
+            python3-requests \
+            curl \
+            sudo \
+            openssl \
+            netcat-openbsd \
+            telnet \
+            llvm-9 \
+            brotli
```

Comment:
please, see the best practices for apt in https://github.com/ClickHouse/ClickHouse/blob/master/docker/test/util/Dockerfile#L62